api/http.go

```diff
@@ -59,15 +59,16 @@ func NewCatalystAPIRouter(cli config.Cli, vodEngine *pipeline.Coordinator, bal b
 	router.GET("/ok", withLogging(catalystApiHandlers.Ok()))
 
 	// Playback endpoint
-	router.GET("/asset/hls/:playbackID/*file",
-		withLogging(
-			withCORS(
-				withGatingCheck(
-					handlers.PlaybackHandler(),
-				),
+	playback := withLogging(
+		withCORS(
+			withGatingCheck(
+				handlers.PlaybackHandler(),
 			),
 		),
 	)
+	router.GET("/asset/hls/:playbackID/*file", playback)
+	router.HEAD("/asset/hls/:playbackID/*file", playback)
+	router.OPTIONS("/asset/hls/:playbackID/*file", withLogging(handlers.PlaybackOptionsHandler()))
```

Comment:
I think we might need the `withCORS` here as well, wdyt?

Reply:
oh yep, think you're right